book_source/installation/thredds.Rmd

```diff
@@ -0,0 +1,341 @@
+# Installing and configuring Thredds for PEcAn
+authors - Rob Kooper
+
+**NOTE: Instructions are only tested for Ubuntu 16.04 on the VM.**
+**NOTE: Pretty much every step here requires root access.**
+
+## Install the Tomcat 8 and Thredds webapp
+
+The Tomcat 8 server can be installed from the default Ubuntu repositories. The thredds webapp will be downloaded and installed from unidata.
+
+## Ubuntu
+
+First step is to install Tomcat 8 and configure it.
+
+```
+apt-get -y install tomcat8 openjdk-8-jdk
+echo JAVA_OPTS=\"-Dtds.content.root.path=/home/carya \${JAVA_OPTS}\" >> /etc/default/tomcat8
+echo JAVA_OPTS=\"-Djava.security.egd=file:/dev/./urandom \${JAVA_OPTS}\" >> /etc/default/tomcat8
+service tomcat8 restart
+```
```

Comment:
@robkooper could you elaborate in comments about what you are doing here?  If/when I need to install on RHEL it would be helpful to know what might need to be different